src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -16,47 +16,63 @@
     {% endset %}
 
     {% set text %}
-        {% if collector.enabled %}
-            {% if collector.token %}
+        {% if collector.impersonated %}
+            <div class="sf-toolbar-info-group">
                 <div class="sf-toolbar-info-piece">
-                    <b>Logged in as</b>
-                    <span>{{ collector.user }}</span>
+                    <b>Impersonating as</b>
+                    <span>{{ collector.impersonatedSourceUser }}</span>
```

Comment:
What about "Impersonator" instead? When reading this (at least as non-native English speaker), it sounds to me as if I'm impersonating as this person, while I am this person, impersonating as someone else.